### DISCUSSION TITLE
Review of draw.io

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#230] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/drawio.svg">

**Comment by eletrofocus** [comment]
Simplesmente um dos melhores programas open source para desenhos tecnicos que já conheci no universo linux.

**Comment by guillaume-mueller** [comment]
Very simple and elegant implementation, but the fact that the inserted diagram is a bitmap ruins it ! Please support vector format or it's 99% useless.

**Reply by Misokados** [reply]
Hello,
we're checking this request. I will contact you as soon as possible.

**Reply by Misokados** [reply]
Dear @guillaume-mueller 
We have added your request to internal tracksystem (74536), we're working on the mentioned feature already.

**Comment by deikatsuo** [comment]
amazing
